editor/import/editor_import_collada.cpp

```diff
@@ -771,7 +771,7 @@ Error ColladaImport::_create_mesh_surfaces(bool p_optimize, Ref<ArrayMesh> &p_me
 
 						int binormal_pos = (binormal_src->stride ? binormal_src->stride : 3) * p.indices[src + binormal_ofs];
 						ERR_FAIL_INDEX_V(binormal_pos, binormal_src->array.size(), ERR_INVALID_DATA);
-						Vector3 binormal = Vector3(binormal_src->array[binormal_pos + 0], binormal_src->array[binormal_pos + 1], binormal_src->array[binormal_pos + 2]);
+						Vector3 binormal = Vector3(-binormal_src->array[binormal_pos + 0], -binormal_src->array[binormal_pos + 1], -binormal_src->array[binormal_pos + 2]); // Due to Godots face order it seems we need to flip our binormal!
```

Comment:
This is essentially the same as I've done in my [colladafix2 branch](https://github.com/Windfisch/godot/tree/colladafix2): The corresponding change is: https://github.com/Windfisch/godot/commit/d34a831bf674f6869e17ab1548090aa10a34963e#diff-f4c0a679eb47b23489fb761cc8c98821R781
Given that two people came to the same conclusion, I think this is right.

Reply:
As mentioned, keep in mind that to counter the problems in the material setting the depth/parallax had the "flip binormal" setting turned ON. This is why QuadMesh looked correct even though it isn't. All these changes are based on not having to flip the binormal